ui/webui/src/components/storage/InstallationDestination.jsx

```diff
@@ -212,8 +213,9 @@ const LocalStandardDisks = ({ deviceData, idPrefix, setIsFormValid, onAddErrorNo
                         usableDisks,
                     });
                     setDisks(usableDisks.reduce((acc, cur) => ({ ...acc, [cur]: defaultDisks.includes(cur) }), {}));
-                }, console.error);
-    }, [refreshCnt]);
+                })
+                .then(refreshDeviceData, console.error);
+    }, [refreshCnt, refreshDeviceData]);
```

Comment:
Student question: this dependency is the reason why the refreshDeviceData is using useCallback, right?

Reply:
Right, if refreshDeviceData was not using useCallback this would be re-triggered unnecessarily.